src/app/cluster/details/cluster/mla/alertmanager-config/alertmanager-config-dialog/component.ts

```diff
@@ -88,14 +106,15 @@ export class AlertmanagerConfigDialog implements OnInit, OnDestroy {
     return encode(this.spec);
   }
 
-  private _edit(alertmanagerConfig: AlertmanagerConfig): void {
-    this._mlaService
-      .putAlertmanagerConfig(this.data.projectId, this.data.cluster.id, alertmanagerConfig)
-      .pipe(take(1))
-      .subscribe(_ => {
-        this._matDialogRef.close(true);
-        this._notificationService.success('Updated the Alertmanager Config');
-        this._mlaService.refreshAlertmanagerConfig();
-      });
-  }
+  // private _edit(alertmanagerConfig: AlertmanagerConfig): void {
+  //   console.log(this.data);
+  //   this._mlaService
+  //     .putAlertmanagerConfig(this.data.projectId, this.data.cluster.id, alertmanagerConfig)
+  //     .pipe(take(1))
+  //     .subscribe(_ => {
+  //       this._matDialogRef.close(true);
+  //       this._notificationService.success('Updated the Alertmanager Config');
+  //       this._mlaService.refreshAlertmanagerConfig();
+  //     });
+  // }
```

Comment:
As above.

Reply:
both are deleted in the last commit 
https://github.com/kubermatic/dashboard/pull/4396/commits/0f1af012b86cce8813b81f6a663c22d920ecc9f7